examples/chunked_example_fast/frontend.lua

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env tarantool
 
 local buffer = require('buffer')
-local msgpack = require('msgpack')
+local msgpack = require('tuple.msgpack')
```

Comment:
However below you still use "skip_header" option. This option isn't supported in 1.10. And seems we need to strip buffer's map header by hands how it was done here: https://github.com/tarantool/tarantool/blob/42c64d06d5d1a3ec937b3c596af083a672a68ad8/src/box/lua/net_box.lua#L635

Also please add a note that in 1.10 "collation_id" should be manually converted to "collation" because "tuple-keydef" module just ignores it.

Here you can see how I solve some compatibility problems: https://github.com/tarantool/crud/blob/8204e0f4ee455f061133140307a596a30f607ada/crud/select/iterator.lua
My approach is ugly but works in general.

Reply:
> However below you still use "skip_header" option. This option isn't supported in 1.10. And seems we need to strip buffer's map header by hands how it was done here: https://github.com/tarantool/tarantool/blob/42c64d06d5d1a3ec937b3c596af083a672a68ad8/src/box/lua/net_box.lua#L635

Thanks for pointing out to `skip_header` option, it now explains me why I see this error while running this example:

```
mr_call
=======
2020-11-01 13:11:00.072 [527] main utils.c:980 E> LuajitError: ...tarantool-merge/.rocks/share/tarantool/tuple/msgpack.lua:147: decode_array_header_compat: unexpected msgpack type '0x81'
```
i.e. we see fixmap header, instead of fixarray. Much thanks!

> Also please add a note that in 1.10 "collation_id" should be manually converted to "collation" because "tuple-keydef" module just ignores it.

I'll take care of that once I'd come to #10 and will adapt merger-test.lua

Reply:
> Also please add a note that in 1.10 "collation_id" should be manually converted to "collation" because "tuple-keydef" module just ignores it.

If it really necessary, I'll implement it right in tuple-keydef.

Reply:
I don't know but at least we should have the same behaviour for our readme and documentation for 1.10 and 2.3+ versions. And if it's not so we should describe a way to get expected behaviour. Currently it's not so. I expect users will just copy-paste this example and then modify for themself (because it's still not so obvious).

Implementation on tuple-keydef level looks appropriate especially it works for built-in key_def in the same way.